### DISCUSSION TITLE
How I can send image with notification?

### DISCUSSION THREAD

**Discussion by webcyst-gursahb** [ctwillie/expo-server-sdk-php#21] [Q&A] [answered]
Hi, after long research I am finally able to send a notification to my app but I want to send an Image with a Notification and I don't know how I can send it. Please Tell Me.

**Comment by ctwillie** [comment, marked as answer]
@webcyst-gursahb This seems to be a limitation of Expo notifications and not an issue related to this package. You can read a little more about this missing feature here: https://expo.canny.io/feature-requests/p/push-notifications-w-images

Hope this helps!
